workflows/git/versions/0.0.1/README.md

```diff
@@ -6,10 +6,129 @@ A set of generics git operations
 
 ## Templates
 
-1. [clone](https://github.com/codefresh-io/argo-hub/blob/main/workflows/git/versions/0.0.1/docs/clone-s3.md) 
+1. [clone-s3](https://github.com/codefresh-io/argo-hub/blob/main/workflows/git/versions/0.0.1/docs/clone-s3.md) 
+2. [clone](https://github.com/codefresh-io/argo-hub/blob/main/workflows/git/versions/0.0.1/docs/clone.md)
+3. [commit](https://github.com/codefresh-io/argo-hub/blob/main/workflows/git/versions/0.0.1/docs/commit.md)
 
 ## Security
 
 Minimal required permissions
 
 [Full rbac permissions list](https://github.com/codefresh-io/argo-hub/blob/main/workflows/git/versions/0.0.1/rbac.yaml)
+
+
+## Examples
+### Clone change commit scenario
```

Comment:
lets be allgiend. we don't put examples in the main READ me. move them to the specific templates

Reply:
Boss. 
a. moved to specific templates
b. I have example which shows how the steps works together, Please advice where to place it.

Reply:
moved into the commit template example